### PR TITLE
HRM-86: Added Position to Applicant

### PR DESCRIPTION
**Pull request by spolischook**
- Added Position Entity
- Added relation to Applicant
- Added PositionFormType
- Added PositionController
- Added navigation
- Added positions demo fixtures
- Added migrations

### CODE REVIEW

**Comment by spolischook** [comment]
@yurio please review this

File: src/Oro/Bundle/HRMBundle/Entity/Applicant.php [hunk 1]
```diff
@@ -579,4 +587,28 @@ public function getAddress()
     {
         return $this->address;
     }
+
+    /**
+     * Set position
+     *
+     * @param \Oro\Bundle\HRMBundle\Entity\Position $position
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Entity/Applicant.php]: Please add Oro\Bundle\HRMBundle\Entity\Position to use block and use short class name to improve readability of the code

Comment (spolischook) [inline src/Oro/Bundle/HRMBundle/Entity/Applicant.php]: ya ya, I remember, just missed it.

File: src/Oro/Bundle/HRMBundle/Form/Type/PositionType.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'oro_hrm_position';
+    }
+}
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Form/Type/PositionType.php]: Please add blank line at the end of document

File: src/Oro/Bundle/HRMBundle/Migrations/Data/Demo/data/position.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    <h3>We offer</h3>
+    <ul>
+    <li>Participation in interesting challenging projects</li>
+    <li>Friendly atmosphere and a professional team</li>
+    <li>Competitive salary</li>
+    <li>Paid vacation and sick leave</li>
+    <li>Free English courses with native speakers</li>
+    <li>Comfortable office close to Lukyanivska metro station</li>
+    </ul>
+    </div>
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Migrations/Data/Demo/data/position.yml]: Please add blank line at the end of document

File: src/Oro/Bundle/HRMBundle/Entity/Position.php [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+     * @ORM\Column(name="description", type="text", length=255, nullable=true)
+     */
+    protected $description;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(type="integer", name="number")
+     */
+    protected $number;
```
Comment (yurio) [inline src/Oro/Bundle/HRMBundle/Entity/Position.php]: What this field for? This name does not tell anything about this. Please try to find the better name for this field.

**Comment by yurio** [comment]
Please add tests for new classes

File: src/Oro/Bundle/HRMBundle/Form/Type/PositionType.php (hunk 2 above)
Comment (spolischook) [inline src/Oro/Bundle/HRMBundle/Form/Type/PositionType.php]: done

File: src/Oro/Bundle/HRMBundle/Migrations/Data/Demo/data/position.yml (hunk 3 above)
Comment (spolischook) [inline src/Oro/Bundle/HRMBundle/Migrations/Data/Demo/data/position.yml]: done

File: src/Oro/Bundle/HRMBundle/Entity/Position.php (hunk 4 above)
Comment (spolischook) [inline src/Oro/Bundle/HRMBundle/Entity/Position.php]: renamed to quantity

**Pull request merged**